.github/workflows/readme.yml

```
@@ -6,58 +6,267 @@ on:
+            if git diff --staged --quiet; then
+              echo "ℹ️ No changes to commit"
+              exit 0
+            fi
+            
+            # Attempt to commit and push with retry logic
+            for attempt in 1 2 3; do
+              echo "🔄 Attempt $attempt to commit and push..."
+              
+              # Fetch latest changes
+              git fetch origin main
+              
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)